packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -196,16 +212,25 @@ export class EcsMetric {
   /**
    * Add custom metric
    */
-  public static custom(value: string) {
-    return new EcsMetric(value);
+  public static custom(value: string, ecsMetricProps?: EcsMetricEnumProps) {
```

Comment:
We do not actually need this static method; it duplicates the constructor logic entirely.